documentation/api/query/examples-pql.markdown

```diff
@@ -27,6 +28,12 @@ Query nodes with `green` in their name.
 nodes { certname ~ 'green' }
 ```
 
+or [regexp features](#pg-regex)
```

Comment:
Is this supposed to be `[regexp features][pg-regex]`?

Reply:
yep, thanks.